js/index.js

```diff
@@ -1,6 +1,39 @@
 // Iteration 1: Names and Input
-//
+var hacker1 = "Duarte";
+console.log(`The driver name is ${hacker1}` );
+var hacker2 = "Alexandre";
+console.log(`The navigator name is ${hacker2}`);
 // Iteration 2: Conditionals
-
+if(hacker1.length >= hacker2.length){
+    console.log(`The driver's as a longest name, it has ${driver1.length} characters.`)
+}else if( hacker1.length <= hacker2.length){
+    console.log(`It seems that the navigator has the longest name, it has ${hacker2.length} characters`)
+}else{
+    console.log(`Wow, you both have equaly long names, ${hacker1.length, hacker2.length} characters.`)
+}
```

Comment:
Using the <= && >= will make your last condition redundant. You should do < / > in this case